README.md

```diff
@@ -143,6 +165,14 @@ parameters:
 
 This is due to the following [Drupal Console issue][11].
 
+## Contributing
+
+Please read [the full documentation](https://github.com/openeuropa/openeuropa) for details on our code of conduct, and the process for submitting pull requests to us.
+
+## Versioning
+
+We use [SemVer](http://semver.org/) for versioning. For the available versions, see the [tags on this repository](https://github.com/openeuropa/drupal_module_template/tags).
```

Comment:
Wrong link, it must refer to media_avportal.

Reply:
Fixed.